### ISSUE TITLE
Is this app still necessary?

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/files_videoplayer#253]
The https://github.com/nextcloud/viewer can show all videos, too.

**Comment by gatsinski** [comment]
According to #201 the video player is deprecated in favour of the viewer app.

**Comment by menaceone** [comment]
The viewer can't play videos in shared links.

**Comment by menaceone** [comment]
yes with thisbdeprecated app. but not with a default Nextcloud installation.
